### ISSUE TITLE
Bug: Comment detection on host line is missing

### ISSUE THREAD

**Issue by Mte90** [goodhosts/cli#6]
If a line include a comment and we are adding to the same ip new domain they are added after the comment example:

```
192.168.50.4 vvv # VAGRANT: db53e7b49dc0f3aa85fa47b3d00a40e6 (default) / e605922e-40d0-4629-9db0-8bf0ce7d3670 vvv.test auto.test tideways.vvv.test xhgui.vvv.test
```

If I do
```
cli r vvv
```
The line is:

```
192.168.50.4 # VAGRANT: db53e7b49dc0f3aa85fa47b3d00a40e6 (default) / e605922e-40d0-4629-9db0-8bf0ce7d3670 vvv.test auto.test tideways.vvv.test xhgui.vvv.test
```

So basically if we are adding or deleting something this changes will happens anyway with the comment but new stuff will be appended after the comment.

**Comment by luthermonson** [comment]
super good catch, ill try and fix this weekend.

**Comment by luthermonson** [comment]
https://github.com/goodhosts/hostsfile/pull/7

**Comment by luthermonson** [comment]
@Mte90 think this is fixed, give v1.0.2 a try, close the issue if you’re good.

**Comment by Mte90** [comment]
confirmed, this fixed the issue
